internify/src/components/molecules/JobPosting.js

```diff
@@ -27,10 +31,12 @@ const JobPosting = (props) => {
     details.candidates,
   ];
 
-  function sendJob(){
-    props.actions.addNewJob(data);
+  function sendJob() {
+    props.actions.addJob({ ...data, dateCreated: Date.now() });
+    props.actions.updateJobsOfUser({ authId: user.uid, jobPostings: [jobId] });
```

Comment:
is this going to re-write the jobId, should we not add a spread operator?

Reply:
I agree, let's look into this later today

Reply:
This is not going to re-write the jobId. The new user action pushes the new jobId to the User store array of job postings